harness/determined/pytorch/dsat/_defaults.py

```diff
@@ -3,6 +3,7 @@
 ALL_SEARCH_METHOD_CLASSES = {
     "random": _dsat_search_method.RandomDSATSearchMethod,
     "binary": _dsat_search_method.BinarySearchDSATSearchMethod,
+    "asha": _dsat_search_method.ASHADSATSearchMethod,
```

Comment:
So, I've changed how this works in my branch https://github.com/determined-ai/determined/pull/6846/files. It's fine if we merge, I just am noting that it's a function now to behave more easily with the typing